fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1738,8 +1738,8 @@ tupleInSubquery
     ;
 
 predicateOperations [ParserRuleContext value]
-    : NOT? IN '(' expressionList ')'                                                      #inList
-    | NOT? IN '(' queryRelation ')'                                                       #inSubquery
+    : NOT? IN '(' queryRelation ')'                                                       #inSubquery
```

Comment:
is this for changing priority?

Reply:
Yes, for modifying priority. select * from tbl where v1 in ((select v2 from tbl2)), this SQL, we hope that the predicate after inPredicate exists in the form of relation instead of expr. This can be consistent with exists to achieve the purpose of simplifying the code.